README.md

```diff
@@ -76,7 +76,7 @@ See the [repository README](//debian.nabijaczleweli.xyz/README) for more informa
 
 ### On Arch Linux
 
-Available from the [community repository](https://archlinux.org/packages/community/x86_64/httplz/) as `httplz`:
+Available from the [extra repository](https://archlinux.org/packages/extra/x86_64/httplz/) as `httplz`:
```

Comment:
This mean the package is in like a more fancy level of repo? :eyes:

Reply:
I think this is just a rename; the old is 404 now.